doc/source/release-notes/release-notes_index.rst

```diff
@@ -6,5 +6,6 @@
    :maxdepth: 1
    :glob:
 
-   Percona-XtraDB-Cluster-5.7.11-4beta-25.14.2
+   Percona-XtraDB-Cluster-5.7.12-rc1-26.16
```

Comment:
Ideally name should be -5rc1 and not -rc1.
Do you check this with Evgeniy.

I see this in version.
[kbauskar@kbauskar-Inspiron-7548 percona-xtradb-cluster] $cat VERSION 
MYSQL_VERSION_MAJOR=5
MYSQL_VERSION_MINOR=7
MYSQL_VERSION_PATCH=12
MYSQL_VERSION_EXTRA=-5rc1